File: RegressionTests/JavaCompiler/JavaSources/TestThrow.java
package dot42.java.test;

import junit.framework.*;

public class TestThrow extends TestCase
{
	public void testSimple1()
	{
		boolean ok = false;
		try
		{
			throw new Exception("Test");
		}
		catch (Exception ex) 
		{
			ok = true;
		}
		assertTrue(ok);
	}
}

File: RegressionTests/Compiler/Sources/TestThrow.cs
using System;
using Junit.Framework;

namespace Dot42.Tests.Compiler.Sources
{
    public class TestThrow : TestCase
    {
        public void testSimple1()
        {
			try
			{
				throw new Exception("Test");
				AssertTrue(false);
			}
			catch (Exception) 
			{
				AssertTrue(true);
			}
        }
    }
}
